xds/internal/client/lds_test.go

```diff
@@ -1764,9 +1764,10 @@ func (s) TestGetFilterChain(t *testing.T) {
 				TransportSocket: &v3corepb.TransportSocket{
 					Name: "envoy.transport_sockets.tls",
 					ConfigType: &v3corepb.TransportSocket_TypedConfig{
-						TypedConfig: &anypb.Any{
-							TypeUrl: version.V3UpstreamTLSContextURL,
-						},
+						TypedConfig: func() *anypb.Any {
+							a, _ := ptypes.MarshalAny(&v3tlspb.UpstreamTlsContext{})
```

Comment:
Check the error (panic if non-nil).  Also, it would clean things up a tad to have a simple function:

```go
func marshalAny(m proto.Message) *anypb.Any {
  // call MarshalAny and panic on non-nil err, return *Any
```